ComputerVision/Rest/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace AnalyzeImage
+{
+    static class Program
+    {
+        // Replace <Subscription Key> with your valid subscription key.
+        const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";
+
+        // You must use the same Azure region in your REST API method as you used to get your subscription keys. 
+        const string uriBase = "https://westus.api.cognitive.microsoft.com/vision/v2.0/analyze";
+
+        static void Main()
+        {
+            Console.WriteLine("Detect objects in images:");
+
+            string imageFilePath = @"Images\sample6.png";
+            string remoteImageUrl = "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample4.png";
+
+            if (File.Exists(imageFilePath))
```

Comment:
Let's be consistent in the Main method. Please use the code similar to the SDK case. I found that much cleaner and easier to read. Please update it for the other Rest samples.

e.g.
            Console.WriteLine("Images being analyzed ...");
            var t1 = AnalyzeFromUrlAsync(computerVision, remoteImageUrl);
            var t2 = AnalyzeLocalAsync(computerVision, localImagePath);

            Task.WhenAll(t1, t2).Wait(5000);
            Console.WriteLine("Press ENTER to exit");
            Console.ReadLine();